chapter_preface/index_vn.md

```diff
@@ -511,7 +511,19 @@ future editions of this book will be able to provide insights
 in places the present edition cannot.
 -->
 
-*dịch đoạn phía trên*
+Hầu hết các phần của cuốn sách đều bao gồm mã nguồn hoạt động được, bởi vì
+chúng tôi tin rằng trải nghiệm học bằng cách tương tác đóng vai trò quan trọng trong học sâu.
+Hiện tại, một số kĩ năng nhất định chỉ có thể được rèn luyện thông qua việc cứ dùng thử và kiểm tra lỗi,
```

Comment:
* "trial and error" là phương pháp "thử và sai"
* "một số kĩ năng nhất định" mình thấy từ kỹ năng có vẻ chưa đúng ý của tác giả. Ở đây đọc có thể hiểu là tác giả muốn nói đến "trực giác mách bảo" thông qua nhiều quá trình thử và sai. Từ gần hơn nữa là từ "kinh nghiệm". Vì vậy mình nghĩ có thể là
```suggestion
Hiện tại, một số kinh nghiệm nhất định chỉ có thể được hình thành thông qua phương pháp thử và sai
```

Reply:
Bạn @Sumo qua bên kia trình báo tên nhé

Reply:
bạn @SumoBBQ chứ không phải @sumo nhé.